c10/util/intrusive_ptr.h

```diff
@@ -720,10 +726,7 @@ namespace intrusive_ptr {
 namespace weak_intrusive_ptr {
 
   inline void incref(weak_intrusive_ptr_target* self) {
-    auto wptr = c10::weak_intrusive_ptr<weak_intrusive_ptr_target>::reclaim(self);
```

Comment:
This change is necessary because ::reclaim would fail if intrusive_ptr_target has never been a weak ptr before. WeakIValue calls incref in the situation we are creating a weak_ptr from a strong pointer when none has existing before. This ensures the method do what it says in its name.